### PR TITLE
Fix SQL bindings for variables containing underscores (Fixes #432)

### LINKED ISSUE #432: Embedded SQL does not allow variables with underscores.
The following embedded SQL code:
```
LET col_val := "Value"
EXEC SQL INSERT INTO table VALUES (:col_val);
```
Results in a compile error:

```
2| EXEC SQL INSERT INTO table VALUES (:col_val);
                                      ~~~~
                                      ^
Error N4305: 2:36 variable not found
```

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

File: t/sql-embed.neon [hunk 1]
```diff
@@ -5,9 +5,9 @@ EXEC SQL CONNECT TO :DatabaseName;
 EXEC SQL DROP TABLE IF EXISTS user;
 EXEC SQL CREATE TABLE user(name STRING PRIMARY KEY, password STRING);
 
-LET name: String := "name"
+LET user_name: String := "name"
```
Comment (ghewgill) [inline t/sql-embed.neon:8]: Don't change this existing test here. Instead, add a few new lines to the bottom of this file that tests the use of an underscore in a parameter value.

**Review by ghewgill — CHANGES_REQUESTED** [review]

File: t/sql-embed.neon (hunk 1 above)
Comment (gitlarryf) [reply t/sql-embed.neon]: Added a new test below the current one.  I only used one parameter with an underscore; should I have used two or more?

Comment (ghewgill) [reply t/sql-embed.neon]: One is sufficient.

**Comment by ghewgill** [comment]
The commit message for this should reference the issue it fixes. (The title of the pull request, where you have mentioned the issue number, means nothing.)

**Pull request merged**